### PR TITLE
HELP-Med show questions

### PR DESCRIPTION
**Pull request by am-bean**
Making the questions appear as soon as the instructions have been read. **This uses a Boolean that seems to be misnamed**. For some reason, whether or not the questions appear is controlled by `isGenerativeContext`. To avoid breaking anything else, I have kept the name as it is, but this seems like something that should be refactored eventually.

### CODE REVIEW

File: frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx [hunk 1]
```diff
@@ -330,7 +330,9 @@ const ChatWithInstructions: FC<
                   <div className="flex items-end justify-end gap-4">
                     <GeneralButton
                       text="I understand"
-                      onClick={() => setReadInstructions(true)}
+                      onClick={() => {setReadInstructions(true);
+                        setIsGenerativeContext(false);}
```
Comment (shincap8) [inline frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx:334]: If an user hasn't click on save and then finish, and clicks on the submit button at the end of the page the interaction with the bot wouldn't be properly saved

**Review by shincap8 — COMMENTED** [review]
Please review comments

File: frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx (hunk 1 above)
Comment (shincap8) [reply frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx:334]: Please remove line 334

**Comment by am-bean** [comment]
Good point. The other changes aren't useful without line 334, so I'll just cancel this PR. It would be nice to be able to show the questions right away because some users keep trying to put their answers in the chatbots, but changing the "Submit" button seems like a bad idea right now, so I'll keep it as is.

**Pull request closed without merging**